WEP0000_wep-proposal.rst

```diff
@@ -0,0 +1,228 @@
+  
+Community review, discussion, and updates
+.........................................
+
+The standard community review period for a WEP is **30 days**. The WEP is reviewed by a process called **lazy consensus**. This means that community members either provide feedback, approve the WEP explicitly, or ignore the WEP. 
+
+If the review period ends without a clear consensus, but further discussion is still needed, the review period might be extended by the WEP team. The review period is never shorter than one month, but other WEPs can be submitted in the meantime, and multiple WEPs might be undergoing a community review simultaneously. 
+
+During the community review period, the WEP goes through one or more iterations of feedback and updates. Community feedback is submitted in the form of comments to the PR, and the WEP author is responsible to update the WEP content in case the incoming feedback is accepted.
+
+All discussion on the WEP must happen on the PR in the form of comments in order to be considered as official feedback on the WEP. Of course external conversations can happen on other platforms, but the official record for a WEP discussion is only on the PR itself. If discussion happens on another platform, we recommend copying the most important points or the results of those discussions into the WEP comments.
+
+You must have a GitHub account to post comments to a PR. If you need help getting started with GitHub, email support@writethedocs and the WTD team will help you set up your account and get familiar with GitHub basics. 
+
+Final review and resolution
+...........................
+
+At the end of the community review period, the WEP is accepted if it meets the minimum criteria and received lazy consensus as described in the previous process stages.
+
+The WEP team reviews the final WEP content and if it is accepted, the team merges the PR, removes the **ready-for-discussion** flag, and moves the WEP to the ``accepted`` directory of the GitHub repository. 
+
+Possible resolutions for a WEP are:
+
+Accepted
+  Reason: Community feedback was completed with lazy consensus achieved, and the WEP content is complete and follows the formatting guidelines. 
+  Action: The WEP team merges the PR and stores the WEP in the ``accepted`` directory.
+
+Rejected
+  Reason: Community feedback led to disagreements about the WEP that cannot be resolved, or the original goal of the WEP is no longer considered an improvement to the community. 
+  Action: The WEP team moves the WEP to the ``rejected`` directory and merges the PR with a description of the rejection decision.
+
+Withdrawn
+  WEP author is no longer interested in pursuing the change, or no progress was made during the pre-review stage before the WEP was sent to community review. 
+  Action: The WEP author moves the WEP moves the WEP to the ``withdrawn`` folder and merges the PR with a description of the withdrawal decision.
```

Comment:
Out of curiosity, why are withdrawn PRs merged at all, instead of closed?

Reply:
I'd assume that it allows folks to see any WEP ideas that were proposed and the discussion leading up to the withdrawal. That meta info could help someone riff of that WEP and come up with an alternative proposal.

Reply:
Yea, I'm not attached to this as a process, since the closed PR's will also be public. I think we mostly stole it from Python, and I don't have a good argument for either approach being better.

Reply:
I vote in favor of keeping withdrawn WEPs in the directory, because it's another (possibly easier) way to access old WEPs not via searching or scanning PRs.